onboarding/service.go

```diff
@@ -10,39 +10,63 @@ import (
 
 	"github.com/influxdata/influxdb/v2"
 	icontext "github.com/influxdata/influxdb/v2/context"
-	"github.com/influxdata/influxdb/v2/kv"
 	"go.uber.org/zap"
 )
 
-type OnboardService struct {
-	service     *Service
-	authSvc     influxdb.AuthorizationService
-	alwaysAllow bool
-	log         *zap.Logger
+var (
+	// ErrOnboardingNotAllowed occurs when request to onboard comes in and we are not allowing this request
+	ErrOnboardingNotAllowed = &errors.Error{
+		Code: errors.EConflict,
+		Msg:  "onboarding has already been completed",
+	}
+)
+
+type TenantService interface {
```

Comment:
Having an exported `TenantService` interface here in the `onboarding` package seems a little strange? Does the tenant package have an interface already?

Reply:
That does make sense! Thanks for the additional info.